<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>620</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>汇资需上传工具V1.0-广州汇法出品</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>661</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>上传参数</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>103</y>
       <width>78</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>每日信号量：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="perDayLimit">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>100</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>1000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>315</x>
       <y>102</y>
       <width>65</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>到期日期：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateLimit">
     <property name="geometry">
      <rect>
       <x>389</x>
       <y>100</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>137</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>信号选择：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>69</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>登录账号：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="account">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>66</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>80</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>系统密码：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>67</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="selectXinhao">
     <property name="geometry">
      <rect>
       <x>119</x>
       <y>133</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="maxVisibleItems">
      <number>11</number>
     </property>
     <property name="minimumContentsLength">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="uploadPath">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>271</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>35</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>上传文件夹：</string>
     </property>
    </widget>
    <widget class="QToolButton" name="selectUploadPath">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>26</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>169</y>
       <width>411</width>
       <height>10</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="progressLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>181</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>等待上传......</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(200, 85, 0);</string>
     </property>
     <property name="text">
      <string>每个表格请严格按照格式：
监测名单、证件号码、类型</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startUpload">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>210</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>开始上传</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>661</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>大EXCEL文件分割</string>
    </property>
    <widget class="QLineEdit" name="excelFile">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>271</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>选择文件：</string>
     </property>
    </widget>
    <widget class="QToolButton" name="selectExcelFile">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>26</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>63</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>限制数量：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="limitCount">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>63</y>
       <width>130</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>5000</string>
     </property>
    </widget>
    <widget class="QToolButton" name="selectSavePath">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>97</y>
       <width>26</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>99</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>保存至文件夹：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startSplitExcel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>139</y>
       <width>113</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>开始分割</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="savePath">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>96</y>
       <width>271</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>88</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(200, 85, 0);</string>
     </property>
     <property name="text">
      <string>分割文件完成后，
会自动设置上传目录
默认保存在程序所在目录</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(200, 85, 0);</string>
     </property>
     <property name="text">
      <string>每个表格标题请严格按照格式：
监测名单、证件号码、类型</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="logShow">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>661</width>
      <height>121</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
